netutils/interface.py

```diff
@@ -172,3 +175,148 @@ def abbreviated_interface_name(interface, addl_name_map=None, addl_reverse_map=N
         raise ValueError(f"Verify interface on and no match found for {interface}")
     # If abbreviated name lookup fails, return original name
     return interface
+
+
+def interface_range_compress(
+    interfaces: Union[str, List[str]] = None, prefix: str = "interface range ", max_ranges: int = 5
```

Comment:
Can we default `interfaces` to an empty string to be in line with our typing? or use Optional?

Reply:
Changed to Optional in [last commit](https://github.com/networktocode/netutils/pull/33/commits/8f9a51aaa0357c92a89eb88a5ae84c2797c2c359#diff-e880d02b86528af00499f4d00b3d768decbd9ff9d6c5921ee90479c468e5e043R181)